packages/common/src/utils/validationUtils.ts

```diff
@@ -236,6 +237,25 @@ function shouldBeDisabled(metadata: CertificateMetadata) {
   return metadata.status === CertificateStatus.LOCKED || metadata.status === CertificateStatus.LOCKED_REVOKED
 }
 
+export function setDisableForChildElement(data: CertificateData, validationResult: ValidationResult) {
```

Comment:
I need to think about this a bit. Especially consider if we can make this more generic. Right now it is hardcoded for ConfiguUeCheckboxMultipleCodes, but the disable validation should generic to be used for other configs.